### PR TITLE
Test/sc 215050/hook e2e tests to ci

### PR DESCRIPTION
**Pull request by charles-testco**
### Summary:
- **What:** `<brief description>`
- **Ticket:** [sc<fill_in_issue_number>](https://app.shortcut.com/genepi/story/<fill_in_issue_number>)
- **Env:** `<rdev link>`

### Checklist:
- [ ] I merged latest `<base branch>`
- [ ] I manually verified the change
- [ ] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/frontend/.eslintrc.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ module.exports = {
       },
     ],
     "prettier/prettier": 0,
+    "@typescript-eslint/no-var-requires": 0,
```
Comment (phoenixAja) [inline src/frontend/.eslintrc.js:57]: do we need to add this?

File: src/frontend/e2e/utils/general.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { faker } from "@faker-js/faker";
+const faker = require("@faker-js/faker");
```
Comment (phoenixAja) [inline src/frontend/e2e/utils/general.ts]: oh i see why you added `@typescript-eslint/no-var-requires` is there a reason why we're doing this syntax and not the above?

**Review by phoenixAja — COMMENTED** [review]
looks good, just have a few qs

**Comment by jgadling** [comment]
LGTM

**Comment by charles-testco** [comment]
Replaced by https://github.com/chanzuckerberg/czgenepi/pull/1363

**Pull request closed without merging**